components/Toggle.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+
+let Sun, Moon;
+
+Sun = Moon = styled.svg`
+  height: 26px;
+  margin-right: 20px;
+  transition: all 0.5s linear;
+`;
+
+export const Toggle = ({ theme, toggleTheme }) => {
+  console.log(theme);
+  return (
+    <div onClick={toggleTheme}>
+      {theme === "light" ? (
+        <Moon
+          xmlns="http://www.w3.org/2000/svg"
+          width="48"
+          height="48"
+          viewBox="0 0 24 24"
+          fill="none"
+          stroke="#212121"
+          strokeWidth="2"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+          className="feather feather-moon"
+        >
+          <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
+        </Moon>
+      ) : (
+        <Sun
```

Comment:
what is this sun and moon? define these svg's like you did in logo and import them here